templates/partials/baseball_stats.html

```diff
@@ -18,7 +18,7 @@ <h3>${{ total_expenditure|format_ballpark_number }}</h3>
 </div>
 <div class="col-md-3 col-6">
   <div class="card-body text-center">
-    <h3>{{ median_salary|format_salary }}</h3>
-    <small class="text-muted">Median salary</small>
+    <h3>{{ median_tp|format_salary }}</h3>
+    <small class="text-muted">Median Total Salary</small>
```

Comment:
Copy editing nitpick: I think of salary as a fixed payment on a set schedule, i.e., appropriate to use for base pay, but probably not for figures including variable extra pay. How about "median total pay"?

Reply:
(Here and in the `payroll_expenditure.html` partial.)